### PR TITLE
Adding the `quest_request_items_locale` table

### PR DESCRIPTION
**Pull request by pangolp**
## Description

Adding table that does not exist.

## Screenshots (if appropriate):
quest_request_items_locale-1
quest_request_items_locale-2

### CODE REVIEW

**Review by Si1ker — APPROVED** [review]

**Comment by Si1ker** [comment]
https://github.com/TrinityCore/TrinityCore/commit/024ae15681736db1adea3ac82f666d725d931e6c

esES/esMX:
https://github.com/TrinityCore/TrinityCore/issues/25805

**Comment by pangolp** [comment]
@FrancescoBorzi this table is the only one missing from the wiki's world database. If you were to revise it, we could have at least all the tables created. Although later, there are some that we have to update. Thanks in advance. Thank you @Si1ker for the feedback.

**Review by Helias — APPROVED** [review]

**Comment by pangolp** [comment]
Thank you @Helias for the feedback.

File: docs/quest_request_items_locale.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+[<-Back-to:World](database-world.md)
+
+**Structure**
+
+| Field               | Type        | Attribute | Key | Null | Default | Extra | Comment |
+|---------------------|-------------|-----------|-----|------|---------|-------|---------|
+| [ID][1]             | int(10)     | unsigned  | PRI | NO   | 0       |       |         |
+| [locale][2]         | varchar(4)  |           | PRI | NO   | NULL    |       |         |
+| [CompletionText][3] | text        |           |     | YES  | NULL    |       |         |
```
Comment (Kitzunu) [inline docs/quest_request_items_locale.md:13]: Default is not NULL for either of them

Comment (pangolp) [reply docs/quest_request_items_locale.md:13]: Ok, now I correct it.

**Comment by pangolp** [comment]
Ready @Kitzunu. The default values are already fixed.

**Review by Kitzunu — APPROVED** [review]

**Comment by pangolp** [comment]
Thanks @Kitzunu

**Pull request merged**